src/test/scala/chiselTests/MixedVecSpec.scala

```diff
@@ -277,4 +277,19 @@ class MixedVecSpec extends ChiselPropSpec with Utils {
       })
     }
   }
+
+  property("MixedVec connections should emit FIRRTL bulk connects when possible") {
+    val chirrtl = ChiselStage.emitChirrtl(new Module {
+      val io = IO(new Bundle {
+        val inMono = Input(MixedVec(Seq(UInt(8.W), UInt(16.W), UInt(4.W), UInt(7.W))))
+        val outMono = Output(MixedVec(Seq(UInt(8.W), UInt(16.W), UInt(4.W), UInt(7.W))))
+        val inBi = Input(MixedVec(Seq(UInt(8.W), UInt(16.W), UInt(4.W), UInt(7.W))))
+        val outBi = Output(MixedVec(Seq(UInt(8.W), UInt(16.W), UInt(4.W), UInt(7.W))))
+      })
+      io.outMono := io.inMono
+      io.outBi <> io.inBi
+    })
+    chirrtl should include ("io.outMono <- io.inMono @[MixedVecSpec.scala")
+    chirrtl should include ("io.outBi <- io.inBi @[MixedVecSpec.scala")
+  }
```

Comment:
It would be nice to see a case where the MixedVecs are parameterized differently. Can they be bulk connected in that case? If not it would be nice to see the check that they are NOT bulk connected.